projects/ngrx.io/content/guide/eslint-plugin/index.md

```diff
@@ -8,6 +8,35 @@ The plugin comes with a number of rules that help address most popular NgRx malp
 
 Some rules also allow automatic fixes with `ng lint --fix`.
 
+## Adding rules
+
+Import the plugin via `plugins` or `extends` to setup the rules via the `rules`
```

Comment:
```suggestion
To use the a [rule](#rules), import the NgRx plugin via `plugins` or `extends`.
You can add a rule by adding the rule to the `rules` collection.

```

Reply:
i agree and added your suggestion in the commit [493e731](https://github.com/ngrx/platform/pull/4123/commits/493e731474ac6431d85a981313cd581dd8ea3c70)